### DISCUSSION TITLE
我想投递游戏rimworld的更新，带DLC

### DISCUSSION THREAD

**Discussion by FortiterInRe** [Gamuxorg/bbs#429] [投递和请求游戏]
种子如下
[rimworld_1_4_3704_rev874_63984.zip](https://github.com/Gamuxorg/bbs/files/14232478/rimworld_1_4_3704_rev874_63984.zip)

**Comment by hyh19962008** [comment]
您方便打成appimage包吗

**Reply by FortiterInRe** [reply]
请问您下载了吗？下载之后会发现其中有4个安装脚本，需手动安装后方可启动，这个我还不会打包成appimage，抱歉了。
